merkle/hstar2.go

```diff
@@ -52,13 +52,22 @@ type SparseSetNodeFunc func(depth int, index *big.Int, hash trillian.Hash) error
 // plus the extra values passed in.
 // It uses the get and set functions to fetch and store updated internal node
 // values.
-func (s *HStar2) HStar2Nodes(n int, values []HStar2LeafHash, get SparseGetNodeFunc, set SparseSetNodeFunc) (trillian.Hash, error) {
+//
+// The treeOffset argument is used when the tree to be calculated is part of a
```

Comment:
This param could arguably be called treeLevelOffset but I wouldn't insist on it.